src/coreclr/gc/gc.cpp

```diff
@@ -43948,6 +43943,26 @@ HRESULT GCHeap::Initialize()
         return CLR_E_GC_LARGE_PAGE_MISSING_HARD_LIMIT;
     }
 
+#ifdef USE_REGIONS
+    gc_heap::regions_range = (size_t)GCConfig::GetGCRegionRange();
+    if (gc_heap::regions_range == 0)
+    {
+        if (gc_heap::regions_range == 0)
+        {
+            if (gc_heap::heap_hard_limit)
+            {
+                gc_heap::regions_range = 2 * gc_heap::heap_hard_limit;
+            }
+            else
+            {
+                gc_heap::regions_range = max(274877906944L, (size_t)(2 * gc_heap::total_physical_mem));
```

Comment:
instead of 274877906944L, should change it to ((size_t)256 * 1024 * 1024 * 1024), much more readable (it's no longer in a macro so don't need to worry about doing computations that might cause some surprises).